File: UETS/app/src/main/java/tr/gov/saglik/uets/api/ApiUtils.java
package tr.gov.saglik.uets.api;


import tr.gov.saglik.uets.api.interfaced.APIService;

/**
 * Created by fatihkocak on 25.04.2019.
 */

public class ApiUtils {
    private ApiUtils() {}

    public static final String BASE_URL = "http://172.18.230.37:83/";
    public static final String BASE_URL_82 = "http://172.18.230.37:82/";

    public static APIService getAPIService() {

        return RetrofitClient.getClient(BASE_URL).create(APIService.class);
    }


    public static APIService getAPIService82() {

        return RetrofitClient.getClient(BASE_URL_82).create(APIService.class);
    }
}

File: KotlinRetrofit/app/src/main/java/tr/com/fkocak/kotlinretrofit/api/ApiUtils.kt
package tr.com.fkocak.kotlinretrofit.api

class ApiUtils {
    companion object {
        private val BASE_URL: String = "https://www.simplifiedcoding.net/demos/"
        fun getAPIService(): ApiService {
            return RetrofitClient.getClient(BASE_URL).create(ApiService::class.java)
        }
    }
}